manifests/charts/base/crds/crd-mixer.yaml

```diff
@@ -18,18 +18,29 @@ spec:
     plural: adapters
     singular: adapter
     categories:
-    - istio-io
-    - policy-istio-io
+      - istio-io
+      - policy-istio-io
   scope: Namespaced
-  subresources:
-    status: {}
   versions:
     - name: v1alpha2
+      schema:
+        openAPIV3Schema:
+          properties:
+            spec:
+              x-kubernetes-preserve-unknown-fields: true
+              type: object
+            status:
+              x-kubernetes-preserve-unknown-fields: true
+              type: object
+          type: object
       served: true
       storage: true
+      subresources:
+        status: {}
+
 ---
 kind: CustomResourceDefinition
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Why are we upgrading Mixer configs to stable v1 when we are about to delete all the code right after 1.7?

Reply:
It is not updating the mixer configs, it is just updating the CRD version. mixer configs still stay at their original version. It is mainly for consistency of CRD versions we use, so that we don't have some CRDs use v1beta1 and some use v1.

Reply:
Ack.